### PR TITLE
Adding README.md to each modules

### LINKED ISSUE #128: Adding README.md for each module
Adding a short brief `REAEME.md` in `src/slam/`, `src/object-recognition` and `src/person-tracking/` dir.

This is required by `npmjs.com`.

### PR DESCRIPTION
**Pull request by kenny-y**
Closes #128 

@tingshao @DonnaWuDongxia @lisa0314 Please help review readme of your module. Thanks.

### CODE REVIEW

File: src/person-tracking/README.md [hunk 1]
```diff
@@ -1,25 +1,43 @@
...
+  });
+});
 ```
-# IDL:
-the code is based on the ./idl/person-tracking.widl, ./idl/common/realsense-camera.widl, ./idl/common/geometry.widl currently.
+
+# Document
+Please refer to the [project document home page](https://01org.github.io/node-realsense/), and also the details page listed below:
+- [Concept and terms](https://software.intel.com/sites/products/realsense/person/developer_guide.html#Introduction)
+- [Object Module document](https://01org.github.io/node-realsense/doc/spec/person-tracking.html)
```
Comment (halton) [inline src/person-tracking/README.md]: Object -> Person

File: src/slam/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  slam.start().then(() => {
+    console.log('Camera + algorithm started...');
+  });
+});
+```
+
+# Document
+Please refer to the [project document home page](https://01org.github.io/node-realsense/), and also the details page listed below:
+- [Concept and terms](https://software.intel.com/sites/products/realsense/slam/developer_guide.html)
+- [Object Module document](https://01org.github.io/node-realsense/doc/spec/slam.html)
```
Comment (halton) [inline src/slam/README.md]: Object -> SLAM

**Review by halton — CHANGES_REQUESTED** [review]
LGTM with two typos.

**Review by halton — APPROVED** [review]
LGTM

**Comment by halton** [comment]
Will merge when trybot green.

File: src/person-tracking/README.md [hunk 3]
```diff
@@ -1,25 +1,43 @@
...
-```
+# Intel® RealSense™ technology with JavaScript API for Linux OS
 
-# How to run samples
-```
-cd src/person-tracking
-npm run maketarball
-```
-The above step generates a person-tracking-xxx.tar.gz file.
+`node-person`: Node.js Person Detection/Person Tracking/Person Recognition/Persong Skeleton Tracking/Body Gesture Recognition/Face Tracking algorithm module based on Intel® RealSense™ technology.
```
Comment (tingshao) [inline src/person-tracking/README.md:3]: currently, PT supports: 
Person Detection/Person Tracking/Person Skeleton Recognition/Body Gesture Recognition

while, Person Recognition and Face Tracking are the same feature and not supported yet. Is it better to remove those 2 ?

**Review by tingshao — COMMENTED** [review]
Thanks for the update, shall we remove unsupported feature and add them when supported?

File: src/person-tracking/README.md (hunk 3 above)
Comment (halton) [reply src/person-tracking/README.md:3]: Is the native documentation mention the supported feature? If so, I would like to point to there

Comment (tingshao) [reply src/person-tracking/README.md:3]: @halton Yes, they are said to be supported. And sorry for my misunderstood of taking Face Tracking and Person Recognition as the same feature, they are not actually. Then, that LGTM.

Comment (halton) [reply src/person-tracking/README.md:3]: So, okay to merge?

Comment (tingshao) [reply src/person-tracking/README.md:3]: LGTM for PT :)

**Pull request merged**